rpm/udisks2.spec

```diff
@@ -1,35 +1,36 @@
-%global glib2_version                   2.36
+%global glib2_version                   2.50
 %global gobject_introspection_version   1.30.0
 %global polkit_version                  0.102
 %global systemd_version                 208
 %global dbus_version                    1.4.0
-%global libblockdev_version             2.14
+%global libblockdev_version             2.25
 
 Name:    udisks2
 Summary: Disk Manager
-Version: 2.8.1
+Version: 2.9.4
 Release: 1
 License: GPLv2+
-Group:   System Environment/Libraries
 URL:     https://github.com/storaged-project/udisks
 Source0: %{name}-%{version}.tar.bz2
 Source1: udisks2-symlink-mount-path
 Source2: udisks2-fs-mount-whitelist.txt
 
-Patch1: 0001-Disable-libblockdev-mdraid-and-part-support-from-sou.patch
-Patch2: 0002-Drop-smartata-dependencies.patch
-Patch3: 0003-Loosen-up-polkit-policies-to-work-from-another-seat.patch
-Patch4: 0004-Introduce-mount-sd-service-that-is-executed-as-user.patch
-Patch5: 0005-Add-udev-rule-for-the-sda-drives.patch
-Patch6: 0006-Disable-zram-rule-for-now.patch
-Patch7: 0007-Create-mount-path-with-755-rights.patch
-Patch8: 0008-Make-it-possible-to-encrypt-mmcblk-format-with-encry.patch
-Patch9: 0009-Reduce-reserved-blocks-percentage-to-zero-for-ext2-e.patch
-Patch10: 0010-Allow-rescan-for-inactive.patch
-Patch11: 0011-Allow-whitelisting-filesystems-that-can-be-mounted.patch
-Patch12: 0012-Add-option-to-set-group-permissions.patch
-Patch13: 0013-Pass-extra-mount-options-to-mount-sd-service.patch
-Patch14: 0014-Always-mount-filesystems-using-the-UUID-instead-of-l.patch
+Patch000: 0001-Make-libblockdev-mdraid-and-part-support-optional.patch
+Patch001: 0002-Make-libatasmart-support-optional.patch
+Patch002: 0003-Loosen-up-polkit-policies-to-work-from-another-seat.patch
+Patch003: 0004-Introduce-mount-sd-service-that-is-executed-as-user.patch
+Patch004: 0005-Disable-zram-rule-for-now.patch
+Patch005: 0006-Create-mount-path-with-755-rights.patch
+Patch006: 0007-Make-it-possible-to-encrypt-mmcblk-format-with-encry.patch
+Patch007: 0008-Reduce-reserved-blocks-percentage-to-zero-for-ext2-e.patch
+Patch008: 0009-Allow-rescan-for-inactive.patch
+Patch009: 0010-Allow-whitelisting-filesystems-that-can-be-mounted.patch
+Patch0010: 0011-Add-option-to-set-filesystem-group-permissions.patch
+Patch0011: 0012-Pass-extra-mount-options-to-mount-sd-service.patch
+Patch0012: 0013-Always-mount-filesystems-using-the-UUID-instead-of-l.patch
+Patch0013: 0014-Check-if-GTK_DOC_CHECK-is-defined-in-case-gtk-doc-is.patch
+Patch0014: 0015-Add-workaround-in-case-gtk-doc-isn-t-installed.patch
```

Comment:
I think there are some extra zeros here. I think it would make sense to have the numbers like this 00, 01, ..., 09, 10, ..., 14.

Reply:
That's to enforce the proper patch order, RPM might patch files in the wrong order without zeros in front.

Reply:
If rpm didn't get numbers right, we'd have breakage for that all over the place.

Reply:
It does sometimes that's why we have the same naming for polkit patches.